### DISCUSSION TITLE
Telegram notifications

### DISCUSSION THREAD

**Discussion by fanynek86** [Ylianst/MeshCentral#6531] [Q&A]
Hello gentlemen!

I have MeshCentral connected to Telegram bot. Is there any way to set a notification to go on Telegram when there is an invalid login attempt?

**Comment by si458** [comment]
nope, however the is a request for this exact thing the other week https://github.com/Ylianst/MeshCentral/issues/6495
so we will look into it when we get chance

**Comment by fanynek86** [comment]
Has anyone ever told you that you are incredibly quick-witted and really have an overview of the situation? :D

Anyway, thank you very much and sorry for the duplicate question.
